### PR TITLE
Bump gke-large-cluster to 2k nodes to experiment

### PR DESCRIPTION
**Pull request by zmerlynn**

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml [hunk 1]
```diff
@@ -266,16 +266,17 @@
...
-                export ZONE="us-central1-b"
-                export NUM_NODES=1025
-                export MACHINE_TYPE="n1-highmem-2"
+                export ZONE="us-east1-a"
+                export NUM_NODES=2000
+                export MACHINE_TYPE="n1-standard-1"
                 export ALLOWED_NOTREADY_NODES="10"
                 export CLUSTER_IP_RANGE="10.8.0.0/13"
-                export GKE_CREATE_FLAGS="--max-nodes-per-pool=100"
+                export GKE_CREATE_FLAGS="--max-nodes-per-pool=100 --no-enable-cloud-monitoring --disable-addons HorizontalPodAutoscaling no-heapster"
```
Comment (wojtek-t) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml]: For curiosity:
1. is HPA running as an addon in GKE? (in OS it's part of controller manager)?
2. I guess "no-heapster" is some GKE-specific flag, right?

**Comment by wojtek-t** [comment]
@zmerlynn - I'm lacking some knowledge of "GKE-creation" flags. I'm pretty sure you figured out exactly what you need, but I just added some question for curiosity.

I will merge on green.

**Comment by zmerlynn** [comment]
@wojtek-t: Yeah, it took me a little iteration to get that right, but I tested before I threw this up. :)

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml (hunk 1 above)
Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml]: Oh, shoot. There's one bit of cruft there. Fingers flying too fast, jussec.

Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml]: HPA is running the same way, we just allow you to flag it off on the CLI. (But you have to disable that *and* monitoring to disable Heapster.)

**Comment by zmerlynn** [comment]
And now I'm in some surreal GitHub state where I've pushed the new changes but they're not showing up.

**Comment by wojtek-t** [comment]
@zmerlynn - I had this problem also at some point; for me fetch + rebase + push helped

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml (hunk 1 above)
Comment (wojtek-t) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml]: OK - makes sense. Thanks for explanation

**Comment by zmerlynn** [comment]
I would, but there's nothing to rebase. The UI delta isn't showing the true/same state of the branch that `git` itself is.

**Comment by wojtek-t** [comment]
@zmerlynn - that LGTM

You have write access to this repository, to self-lgtm and self-merge it when you will be ready.

**Comment by zmerlynn** [comment]
Cool, will do.

**Pull request merged**